### PR TITLE
[WIP] systemd-bootchart part 2

### PR DESCRIPTION
**Pull request by akavel**
###### Motivation for this change

Extend PR #16902 with patches fixing broken rendering of the processes graph in systemd-bootchart.

(I'm OK if my commit is moved to @abbradar's PR and "my" PR then closed; didn't know how to submit the commit otherwise on top of the previous PR.)
###### Things done
- [ ] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/os-specific/linux/systemd/bootchart.nix [hunk 1]
```diff
@@ -21,6 +21,13 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ systemd ];
 
+  patches = [
+    # https://github.com/systemd/systemd-bootchart/pull/10
+    ./bootchart-fix-processes-graph.patch
```
Comment (abbradar) [inline pkgs/os-specific/linux/systemd/bootchart.nix]: It's better to use `fetchpatch` there -- we try not to store any patches in the tree. For example, the first patch's URL is https://github.com/akavel/systemd-bootchart/commit/c5e0f0e2669b048502fdd9c4d1116bf2fb22c20f.patch

**Comment by Ericson2314** [comment]
@akavel you can submit a PR against the branch the original PR was made from, and if the owner of that branch accepts your PR, the original PR will be automatically updated.

**Comment by abbradar** [comment]
Hm, I thought about this but the downside is that there might be other potential reviewers and interested parties and making a PR to my fork hides it from them. Maybe it's overthinking on my part, though...

**Comment by Ericson2314** [comment]
That's perfectly fine. @akavel said they didn't know how, so I just provided the information.

**Comment by mmahut** [comment]
Are these any updates on this issues?

**Comment by mmahut** [comment]
Closing due to lack of activity, feel free to reopen this if needed.

**Pull request closed without merging**